anda/tools/nerdfetch/nerdfetch-tools.spec

```
@@ -0,0 +1,33 @@
+Name: nerdfetch
+Version: 8.4.0
+Release: 1%?dist
+Summary: A POSIX *nix fetch script using Nerdfonts 
+
+License: MIT
+URL: https://github.com/ThatOneCalculator/NerdFetch
+Source0: %url/archive/refs/tags/v%version.tar.gz
+
+BuildArch: noarch
+Requires: bash
```

Since this can require any shell the recommended way to dep this would be a file dep:
```suggestion
Requires: /usr/bin/sh
```